### PR TITLE
Add a Telemetry schema for Firefox Voice

### PR DESCRIPTION
**Pull request by ianb**
This is a schema for a custom Telemetry ping for the Firefox Voice project (an extension that receives and handles voice commands).

We talked with @sunahsuh about this, but I'm kind of making it up as I go, so I'm not sure if I've put the schema in the appropriate location, or if there's other things I should include in the PR.

Note this includes PII (the exact voice commands), and it's our understanding that it's OK to store this information in Telemetry (assuming that we've received appropriate consent in-product). We've avoided full URLs, though we have included URL domains.

Note: we're still getting data review on this, and will hold the PR until we've also sent this proposed data through that review.

The actual data:
* We expect to submit one ping per interaction, so this covers everything from starting the voice interaction, speaking, translating that to text, interpreting the text, and executing the action.
* I've tried to mostly make everything flat, and we'll null out everything that doesn't apply to a specific interaction.
* We'll do full error handling through Sentry, but we include some brief indication of errors in the ping.
* We'll null out information if we don't have consent for that category of information. But I haven't included any properties to indicate the consent. (Should I?)
* I've included everything I think we might want to collect, but I expect we won't instrument everything right away. I'm assuming it's easier to include it all now.
* There's some `"object"` types where specific intent handlers can save arbitrary information. I expect we'll mostly want to download and process this information offline. Is that reasonable? Should we force these arbitrary bits of information into a string?
* **Question:** we want to get thumbs up/down reactions from users for a specific interaction, but that rating will happen after the ping has been submitted. How can we collect that information and relate it to the original interaction? E.g., we could do a separate ping type, and include IDs in both this ping and that ping to relate them. Or can we amend a submitted ping? Or use one kind of Telemetry ping, and use timestamps to see how they relate?

----

Checklist for reviewer:

- [ ] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [ ] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional)

For glean changes:
- [ ] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

File: templates/telemetry/voice/voice.1.schema.json [hunk 1]
```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "voice",
+  "properties": {
+    @TELEMETRY_APPLICATION_1_JSON@,
+    @TELEMETRY_CLIENTID_1_JSON@,
+    @TELEMETRY_CREATIONDATE_1_JSON@,
+    @TELEMETRY_ENVIRONMENT_1_JSON@,
+    @TELEMETRY_ID_1_JSON@,
```
Comment (ianb) [inline templates/telemetry/voice/voice.1.schema.json]: I just copied these from the Test Pilot ping, I don't really understand what they are, but I assume this is all the "standard" environment values.

File: templates/telemetry/voice/voice.1.schema.json [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+  "required": [
+    "application",
+    "clientId",
+    "creationDate",
+    "environment",
+    "id",
+    "payload",
+    "type",
+    "version"
+  ]
```
Comment (ianb) [inline templates/telemetry/voice/voice.1.schema.json]: Again I assume these are from those variable substitutions above

**Comment by ianb** [comment]
I see a test failed, but I don't see anything that I understand as a failure in the output...?

**Comment by sunahsuh** [comment]
Hey @ianb, looks like you're missing the generated schema per: https://github.com/mozilla-services/mozilla-pipeline-schemas#adding-a-new-schema

I'm tagging in @acmiyaguchi since I'm out this week (please redirect elsewhere, Anthony, if you don't have bandwidth to take a look)

**Comment by ianb** [comment]
I've added the generated schema and a parque schema

**Comment by ianb** [comment]
@acmiyaguchi @sunahsuh We had some delays on privacy review, but now that we're past that, here is a version of the patch that we are ready to land.

**Comment by ianb** [comment]
We don't know a lot about the querying, and we don't have any particular expectations. I put in the parquet spec out of thoroughness more than anything. So whatever happens with that, so it goes ;)

**Comment by ianb** [comment]
I assume from the approval that it's ready to merge... so I have made that assumption and merged it!

**Comment by sunahsuh** [comment]
Fantastic -- the schemas and tables should be deployed to production in a few hours

**Pull request merged**